packages/introspect/src/generate.ts

```diff
@@ -17,18 +17,24 @@ export async function generate({
     defaultSchema,
     outDir,
     formatter = async (str) => str,
+    config = {
```

Comment:
don't wrap this in an additional `config` object. The wrapping is adding no additional value. The config object is already wrapping all the properties

Reply:
Also note that there is actually a bug here: you can pass `config = {}` which will not default to config = {importLocation}

Reply:
Given that this should not be used by normal users I would suggest using a more explicit name like`dangerous_importQueriesLocation`